src/Authentication/AuthProvider.jsx

```diff
@@ -0,0 +1,57 @@
+import { createContext, useEffect, useState } from 'react';
```

Comment:
Use shortcut-key/UI to fix the indentation of the code.
For example: 
 const [loading, setLoading] =useState(true)  needs to be const [loading, setLoading] = useState(true);